src/components/v5/common/Modals/ManageMemberModal/ManageMemberModal.tsx

```diff
@@ -44,6 +54,44 @@ const ManageMemberModal: FC<ManageMemberModalProps> = ({
         placeholderText={{ id: 'members.modal.selectActions' }}
         isListRelative
       />
+
+      {(isBanOptionSelected || isUnbanOptionSelected) && (
+        <>
+          <div className="border border-gray-300 bg-base-bg rounded p-4 text-gray-600 text-sm mt-4">
+            <span className="font-medium">
+              {formatMessage({ id: 'note' })}:{' '}
+            </span>{' '}
+            <span>
+              {formatMessage({
+                id: isBanOptionSelected
+                  ? 'members.ban.info'
+                  : 'members.unban.info',
+              })}
+            </span>
+          </div>
+
+          <div className="text-gray-700 text-1 flex justify-between my-4">
+            <div className="flex items-center gap-2">
+              {formatMessage({ id: 'members.ban.notify' })}
+              <span className="text-gray-400 flex">
+                <Icon name="info" appearance={{ size: 'extraTiny' }} />
+              </span>
+            </div>
+            <Switch />
+          </div>
+
+          <Textarea textareaTitle={{ id: 'members.ban.textarea.label' }} />
+
+          <div className="flex mt-8 gap-2">
```

Comment:
As I can see we have a login to Cancel and Confirm buttons in the Modal component. There are props closeMessage and confirmMessage. Please, use them to be consistent and not repeat buttons and their styles.